src/RemotePullManager.php

```diff
@@ -160,19 +179,16 @@ public function isAutopullNeeded(Remote $remote, $dry_run = FALSE) {
 
     // If autopull was never run or the intervals has been reached, we pull.
     if (!$last_autopull || ($last_autopull + $autopull_interval) < time()) {
-      // Don't process the pull on dry run.
-      if ($dry_run) {
-        return TRUE;
+      if (!$dry_run) {
+        // Set the curent time as last pull time.
+        $this->state->set($remote_state_id, time());
+        $this->logger->notice('Last autopull time set to @date', ['@date' => date('m/d/Y H:i:s', time())]);
```

Comment:
That's not really valuable information - that happens always when a pull is triggered. Let'S just drop this line.